### DISCUSSION TITLE
Who is consuming WZDx?

### DISCUSSION THREAD

**Discussion by barbeau** [usdot-jpo-ode/jpo-wzdx#122] [General]
Hi all!

It's been great to see the ongoing evolution of WZDx in recent months, and to see producers actively producing feeds (including Maricopa County DOT, Iowa DOT, massDOT, etc. - feel free to comment if I missed one!).

One recurring question that I've heard is "who is consuming WZDx"?

That's often the chicken/egg question with new cities/counties/DOTs producing data (for any data format) - often producers don't want to produce data until there is someone consuming it, and consumers don't want to implement the software until there is data.

So - are there known consumers of WZDx feeds, especially regionally or nationally that would cover more than one city?

Any info is appreciated! Thanks in advance!

**Comment by lynnerandolph** [comment]
TxDOT is also producing a feed.

I know Drivewyze is consuming the Maricopa County data. They talked to TxDOT about adding their data as well.

Lynne
